### DISCUSSION TITLE
Prevent accepting the first suggestion on enter

### DISCUSSION THREAD

**Discussion by boyeln** [neoclide/coc.nvim#2965] [Q&A] [answered]
Hi :wave: 
In vscode there is a setting called "Accept Suggestion On Enter" which I like having set to false. Resulting in that suggestions only are accepted if I first press `tab` to select a suggestion (when a suggestion is selected, enter should accept it). Is there a way to achieve this with coc?

**Comment by chemzqm** [comment, marked as answer]
Use `complete_info`, read https://github.com/neoclide/coc.nvim/wiki/Completion-with-sources#use-cr-to-confirm-completion

**Reply by boyeln** [reply]
Thanks! Works exactly as I want it to! Should've read the docs more carefully.

**Reply by maximiliandaschner** [reply]
Sorry, it's not quite clear to me what you need to do for this to work. I tried to implement each of the four options given in the link but none of them changed the behaviour of the suggestion box. It still selects the first option on Enter, instead of just creating a new line, which is what I hoped it would do.
